components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -596,21 +598,81 @@ export const StyledCustomBackgroundOption = styled('button')<{}>`
   }
 `
 
-export const StyledUploadIconContainer = styled('div')<{}>`
+interface SelectionProps {
+  selected: boolean
+}
+
+interface SolidColorBackgroundProps {
+  color: string
+}
+
+interface ImageBackgroundProps {
+  image: string
+}
+
+export const StyledSelectionBorder = styled('div')<SelectionProps>`
+  position: relative;
+  width: 100%;
+  height: 160px;
+  ${p => p.selected && css`
+    background: ${p => isDarkTheme(p)
+        ? 'linear-gradient(314.42deg, #FA7250 6.04%, #FF1893 44.31%, #A78AFF 100%)'
+        : 'linear-gradient(122.53deg, #4C54D2 0%, #BF14A2 56.25%, #F73A1C 100%)'};
+    padding: 2px;
+    border-radius: 10px;
+    &:after {
+      content: url(${CheckedCircle});
+      position:absolute;
+      top: 10px;
+      right: 10px;
+      z-order: 1;
```

Comment:
I haven't come across `z-order` before. Should this be `z-index` or is it something new?